### PR TITLE
chore: bump open-aea and plugins versions to latest

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

Bumps to latest

## Fixes

-

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

-

### CODE REVIEW

File: scripts/RELEASE_PROCESS.md [hunk 1]
```diff
@@ -27,18 +27,3 @@
 11. Open PRs and merge into develop. Then open develop to main PR and merge it.
 
 12. Tag version on main.
-
-13. Pull main, make a clean environment (`pipenv --rm` and `pipenv --python 3.10` and `pipenv shell`) and create distributions: `make dist`.
```
Comment (DavidMinarsch) [inline scripts/RELEASE_PROCESS.md:31]: Replaced with `.github/workflows/release.yml`

File: tox.ini [hunk 2]
```diff
@@ -76,21 +76,21 @@ setenv =
 
 [commands-framework]
 commands =
-    autonomy init --reset --author CI --remote --ipfs --ipfs-node "/dns/registry.autonolas.tech/tcp/443/https"
+    autonomy init --reset --author ci --remote --ipfs --ipfs-node "/dns/registry.autonolas.tech/tcp/443/https"
```
Comment (DavidMinarsch) [inline tox.ini:79]: Breaking in 1.28.0

File: deployments/Dockerfiles/autonomy/requirements.txt [hunk 3]
```diff
@@ -1,8 +1,7 @@
 watchdog==2.1.6
 requests==2.27.1
-open-aea[all]==1.27.0.post1
-open-aea-cli-ipfs==1.27.0.post4
-open-aea-ledger-ethereum==1.27.0
-open-aea-ledger-cosmos==1.27.0
-open-aea-ledger-fetchai==1.27.0
```
Comment (DavidMinarsch) [inline deployments/Dockerfiles/autonomy/requirements.txt:7]: Not needed afaik.

File: plugins/aea-test-autonomy/setup.py [hunk 4]
```diff
@@ -36,6 +36,8 @@
     author="Valory AG",
     license="Apache-2.0",
     description="Plugin containing test tools for open-autonomy packages.",
+    long_description="Plugin containing test tools for open-autonomy packages.",
+    long_description_content_type="text/markdown",
```
Comment (DavidMinarsch) [inline plugins/aea-test-autonomy/setup.py:40]: Needed for the workflow to work

File: deployments/Dockerfiles/autonomy/requirements.txt (hunk 3 above)
Comment (DavidMinarsch) [reply deployments/Dockerfiles/autonomy/requirements.txt:7]: (Also `open-aea-ledger-fetchai` needs a post release due to an error in the dependency version definition.

**Comment by DavidMinarsch** [comment]
Test failures:

1. Windows failures: I cannot see any relevant changes [here](https://github.com/valory-xyz/open-aea/compare/v1.27.0.post1...valory-xyz:open-aea:v1.28.0) which should cause this. Not clear yet...
2. Scaffold test failures:
```
tests/test_autonomy/test_cli/test_scaffold_fsm.py::TestScaffoldFSMAutonomyTests
```
I think this is related to [changes in how we execute tests now](https://github.com/valory-xyz/open-aea/compare/v1.27.0.post1...valory-xyz:open-aea:v1.28.0#diff-cd6500f5ac7cb80e821b7b0c2a35d7bd92ae8187b8e1a78998c973d9308962c2R511) - and has uncovered a previously hidden issue.
3. E2E test failures:
Not 100% sure, but the disconnection issues with the multiplexer could be due to the added [locks](https://github.com/valory-xyz/open-aea/compare/v1.27.0.post1...valory-xyz:open-aea:v1.28.0#diff-b6154e70dee53bf5edfe808a6264300cd61fa86a8eea95d4b647c5e5ee63708fR50), preventing disconnect?

**Comment by angrybayblade** [comment]
Regarding the windows dependency issue, you can see the reason from the logs 

```
The conflict is caused by:
    The user requested pywin32==304
    docker 6.0.0 depends on pywin32>=304; sys_platform == "win32"
    open-aea[all] 1.28.0 depends on pywin32==300
```

On open-aea we set `pywin32` to be `==300` if python version is less than or equal to `3.9` and `==304` for py3.10.

Docker v6.0.0 requires `pywin32>=304` (https://github.com/docker/docker-py/blob/6.0.0/setup.py#L21) 

On open-autonomy, we install `pywin==304` in all versions of the python in the CI

https://github.com/valory-xyz/open-autonomy/actions/runs/3918264149/jobs/6698593155#step:12:16
https://github.com/valory-xyz/open-autonomy/actions/runs/3918264149/jobs/6698593170#step:12:16
https://github.com/valory-xyz/open-autonomy/actions/runs/3918264149/jobs/6698593188#step:12:16
https://github.com/valory-xyz/open-autonomy/actions/runs/3918264149/jobs/6698593206#step:12:16

Yet we did not have any conflict till now which makes no sense since open-aea required `pywin32==300` for python versions <3.10

This PR tries fix this issue by pinning `pywin32` to `>=304` since it supports Python >= 3.6 https://github.com/valory-xyz/open-aea/pull/544

**Comment by 0xArdi** [comment]
The E2E failure is due to a merge error and is resolved here https://github.com/valory-xyz/open-autonomy/pull/1747

**Review by Adamantios — APPROVED** [review]

**Pull request merged**